packages/apps/human-app/frontend/src/api/services/worker/oracles.ts

```diff
@@ -24,6 +26,7 @@ const H_CAPTCHA_ORACLE: OracleSuccessResponse = {
   jobTypes: env.VITE_H_CAPTCHA_ORACLE_TASK_TYPES,
   role: env.VITE_H_CAPTCHA_ORACLE_ROLE,
   url: env.VITE_H_CAPTCHA_ORACLE_ANNOTATION_TOOL,
+  registrationNeeded: false,
```

Comment:
Technically, registration is needed for hcaptcha labeling but I think we can skip that for now.

Reply:
the process is just different that's why I set false.